app/src/main/res/drawable/circular_progress_drawable.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<rotate xmlns:android="http://schemas.android.com/apk/res/android"
+    android:fromDegrees="145"
```

Comment:
This sets the progress bar to start at 145 degrees and never move as values are updated.